frameworks/cassandra/universe/marathon.json.mustache

```diff
@@ -129,6 +130,7 @@
     "TASKCFG_ALL_REMOTE_SEEDS": "{{service.remote_seeds}}",
     "EXECUTOR_URI": "{{resource.assets.uris.executor-zip}}",
     "BOOTSTRAP_URI": "{{resource.assets.uris.bootstrap-zip}}",
+    "AWSCLI_URI": "{{resource.assets.uris.awscli-zip}}",
```

Comment:
Should probably just refer to it as the S3CLI throughout, as the aws cli is something else.